### PR TITLE
Changed misspelled "exept" to accept

### PR DESCRIPTION
**Pull request by Boen-Kelly**

### CODE REVIEW

**Comment by democat3457** [comment]
While you're at it, can you also fix 
> TriggerReader class
A class that represents a gamepad trigger
Includes *simlular state - changing* methods like GamepadButton

**Comment by pranavavva** [comment]
@Boen-Kelly can you provided a status update on this?

**Comment by Boen-Kelly** [comment]
@pranavavva What do you need me to provide a status update of?

**Comment by pranavavva** [comment]
Did you make the change that democat suggested above?

**Comment by Boen-Kelly** [comment]
No, what is the problem with it?

**Comment by democat3457** [comment]
* *simlular state - changing*
* *differnt*

**Comment by Boen-Kelly** [comment]
Got it, updated

**Comment by democat3457** [comment]
No I think it's supposed to be 
*simular state-changing methods*   
Also fix `differnt` which is supposed to be `different`

**Comment by Boen-Kelly** [comment]
ok, sorry

File: readme.md [hunk 1]
```diff
@@ -70,7 +70,13 @@ __Features__:
         + Many uses including the current state, the recent state, and more
     + TriggerReader class
         + A class that represents a gamepad trigger
-        + Includes simlular state - changing methods like GamepadButton
+<<<<<<< HEAD
```
Comment (democat3457) [inline readme.md]: Oops there's a merge conflict here apparently

Comment (Boen-Kelly) [reply readme.md]: What would cause that?

Comment (democat3457) [reply readme.md]: probably if the test branch had old stuff that wasn't pulled before pushing

File: readme.md [hunk 2]
```diff
@@ -70,13 +70,7 @@ __Features__:
         + Many uses including the current state, the recent state, and more
     + TriggerReader class
         + A class that represents a gamepad trigger
-<<<<<<< HEAD
         + Includes simlular state-changing methods like GamepadButton
```
Comment (democat3457) [inline readme.md]: Simular, not simlular.

**Comment by Boen-Kelly** [comment]
I'm so sorry. I looked through the top part of the readme.md file, and changed a few things.

**Comment by democat3457** [comment]
Ah its fine.

**Pull request merged**